string tagid
---
geometry_msgs/PoseStamped best_vantage